source/gui/frmhelpgeneratorunit.lfm

```diff
@@ -1,18 +1,17 @@
 object frmHelpGenerator: TfrmHelpGenerator
```

Comment:
Can you describe what has changed, what I should look for when testing this change visually? 

You mention in PR "Made little cosmetic changes and fix layout in pasdoc gui with Lazarus" -- please expand this description :)

Reply:
Just some minor change,  i fix some component to alCustom instead alNone and fix the with for prevent the automatic resizing of the form

Reply:
Hi, no problem, i'm busy too, i'll make the changes soon as possible. Surely this week-end ;)

Cheers